src/segment-loader.js

```diff
@@ -1557,7 +1558,7 @@ export default class SegmentLoader extends videojs.EventTarget {
     const videoSegmentTimingInfoCallback =
       this.handleVideoSegmentTimingInfo_.bind(this, segmentInfo.requestId);
 
-    this.sourceUpdater_.appendBuffer({type, bytes, videoSegmentTimingInfoCallback}, (error) => {
+    this.sourceUpdater_.appendBuffer({segmentInfo, type, bytes, videoSegmentTimingInfoCallback}, (error) => {
```

Comment:
I was actually thinking that we may want to remove logging from `appendBuffer` and just log here instead, I guess this works too.

Reply:
I think it's useful logging directly in the action because then you'll know exactly when it happens instead of just that it was queued.

Reply:
you might want to pass just the important bits of segmentInfo here as it can be quite large. Otherwise LGTM

Reply:
Theoretically, it shouldn't be an issue and could be useful when debugging. I think for this PR, I'd prefer to leave as-is, but we do have memory/performance related work that we're going to do.